### PR TITLE
Testing to do list

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this milestone, we did the following activities:

-Instal Jest with the project 
-create a Pures function and Separate it into different modules 
-test the functions add and remove 
-test the Local Storage Function 
-Test de clear All complete Function 
-Edit linters Files

## Thanks for reviewing our code! 🙌

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @cvalencia1991, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you did alone but this is pair programming. So I would suggest you please implement the project requirement with your partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/__test__/TodoList.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    const list = document.querySelectorAll('#list li');
+    expect(list).toHaveLength(1);
+  });
+});
+
+describe('Checkbox', () => {
+  test('Expect to change completed to tru after click', () => {
+    clearallsuccess();
+  });
+});
```
Comment (t-yanick) [inline src/__test__/TodoList.test.js]: - [ ] As they are [mandatory requirements ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_2.md)for this project, 

Create a test file(s) ([..].test.js) for a file(s) containing the following functions:
 - a function for editing the task description.
 - the "Clear all completed" function.

Please kindly add the above mentioned tests

**Review by t-yanick — APPROVED** [review]
Hi @cvalencia1991, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/TodoList.test.js (hunk 1 above)
Comment (t-yanick) [reply src/__test__/TodoList.test.js]: Hello @cvalencia1991 , @OsvaldoBC ,  please  I mistakenly approved this project, please make sure you implement the requested changes and resubmit for another review. I will be marked as require changes on your dashboards. ***Please do not merge this PR***

**Review by rustam-tolipov — APPROVED** [review]
## Hello Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

No Linter errors 💯

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**